src/pages/HomePage.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const HomePage = () => {
+  const usernameHandler = (event) => {
+    if (event.key === "Enter") {
+      localStorage.setItem("userName", event.target.value);
+      window.location.reload(false);
```

Comment:
can't we use ` event.preventDefault() ` here?

Reply:
no, that works with form only ig.